python/processors/Stop0l_postproc_QCD.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+import os, sys
+import ROOT
+ROOT.PyConfig.IgnoreCommandLineOptions = True
+from importlib import import_module
+from PhysicsTools.NanoAODTools.postprocessing.framework.postprocessor import PostProcessor
+from PhysicsTools.NanoSUSYTools.modules.eleMiniCutIDProducer import *
+from PhysicsTools.NanoSUSYTools.modules.Stop0lObjectsProducer import *
+from PhysicsTools.NanoSUSYTools.modules.Stop0lBaselineProducer import *
+from PhysicsTools.NanoSUSYTools.modules.qcdSmearProducer import *
+#from PhysicsTools.NanoSUSYTools.modules.genjettest import *
+
+#from PhysicsTools.NanoSUSYTools.modules.qcdskimmingfile import *
+mods = [
+    #qcdskimmingfile(),
```

Comment:
You don't need skimming?

Reply:
No, we don't need skimming. But it's commented out.